proxy/proxy.go

```diff
@@ -327,7 +356,15 @@ func (p *Proxy) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 				Tracers:       tracers,
 			}
 
-			_, err = common.AddTracer(r)
+			//_, err = common.AddTracer(r)
+			jsonData, _ := json.Marshal(r)
+
+			req, err := http.NewRequest("POST", "http://"+configure.Current.TracyServer.Addr()+"/api/tracy/tracers", bytes.NewBuffer(jsonData)) //Change this over to https when we do that. Well it would be better to make it a config
+			req.Header.Set("Content-Type", "application/json")
```

Comment:
might be nice to have a method for all boilerplate stuff API and you just pass the route and body or something